### ISSUE TITLE
[Metric Extraction] Agent nagios_perfdata transformer

### ISSUE THREAD

**Issue by nikkiattea** [sensu/sensu-go#1385]
Child of https://github.com/sensu/sensu-go/issues/745
Depends on #1383

Create a function `Transform(string) []MetricPoint, error` from the interface `Transformer` for nagios_perfdata. The function should accept a string (check output) and parse/transform the metrics to Sensu Metric Format. Return an error if the line is incorrectly formatted. The `nagios_perfdata` transformer will implement the [Nagios Perf Data](https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/perfdata.html) line protocol, a single line of human readable text, to extract the MetricPoints:

```
PING ok - Packet loss = 0%, RTA = 0.80 ms
```

or

```
PING ok - Packet loss = 0%, RTA = 0.80 ms | percent_packet_loss=0, rta=0.80
```

Additionally, add agent logic in check_handler.go to implement this transformer if the check’s `metric_format` matches the const `nagios_perfdata`.
